packages/vue/package.json

```diff
@@ -98,5 +98,8 @@
     "*.vue",
     "*.scss",
     "*.css"
-  ]
+  ],
+  "storybook": {
+    "url": "https://docs.storefrontui.io/"
```

Comment:
Have you tested it? I followed the tutorial however it did not work exactly as I wanted (not accordingly to the tutorial screenshots). Only when I added this `refs` rule to main.js the external link appeared. 
Also the branch is outdated. Please update it :)

Reply:
No I didn't. I 'll try it. thanks

Reply:
@Baroshem  Locally it works, but it should, also without added link. But I don't know how to test it on published version. Shoudn't it be this hotfix released first and docs published to see if it works? Or am I wrong?

Reply:
It looks like adding this link will cause that our storybook will appear automatically on every storybook run on projects that have SFUI in dependecies. I don't know if it is desired behaviour, wdyt (although it can be disbaled in main.js file)? Other thing is that we need to generate stories.json file to use storybook composition and it is created by `npx sb create` command (https://dlim716.medium.com/storybook-composition-setup-9e078c590979). It should be included in deployment process to run before every new version of storybook is released.